cfgov/templates/wagtailadmin/login.html

```diff
@@ -17,19 +16,24 @@
             </div>
         {% endif %}
 
-        {% if 'build' in request.site.hostname %}
-            <h1>Log in to demo Wagtail</h1>
-        {% else %}
-            <h1>Log in to consumerfinance.gov</h1>
-        {% endif %}
-        <form action="{% url 'cfpb_login' %}" method="post" autocomplete="off">
+        {% block above_login %}
```

Comment:
This logic feels outdated given the number of Wagtail environments we have. "build" isn't really a good indicator for "demo Wagtail", and non-"build" doesn't necessarily mean "consumerfinance.gov".

Reply:
I think this has to be a larger discussion. I recall this being explicitly added this way. @schaferjh?

Reply:
@willbarton @chosak it probably is outdated but I agree it was added purposefully back in the day. Just jumping in like this, I'm not sure of the consequences of removing it. Happy to have a quick chat if you're trying to move quickly. FYI @ascott1

Reply:
@schaferjh thanks; it's definitely not urgent to remove. I'd be curious if the goal was more to warn people who might be editing cf.gov, or give them assurance that they're on a "demo" system if on build. For example, someone logging into beta or a demo dev server would see the "consumerfinance.gov" message which wouldn't be accurate.

Reply:
@chosak that rings a bell. I'm sure the product owner (!!) asked to make it clear to users logging in to build that they were on a demo system and not editing the live site.